lib/Github/Api/Apps.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Github\Api;
+
+/**
+ * @link   https://developer.github.com/v3/apps/
+ * @author Nils Adermann <naderman@naderman.de>
+ */
+class Apps extends AbstractApi
+{
+    /**
+     * @deprecated
```

Comment:
Oh I see. Move this method back to the Integrations class. It is weird to add a class with deprecated functions.